mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -739,6 +745,9 @@ void mav_vehicle::takeoff()
 
 void mav_vehicle::rotate(double angle_deg, bool autocontinue)
 {
+    // We have taken control
+    this->is_our_control = true;
```

Comment:
Ok, I got the idea. Still looks weird, but I think we can live with that for now.